pkg/relay/ethereum/chain.go

```diff
@@ -321,13 +321,15 @@ func (c *Chain) QueryUnfinalizedRelayPackets(ctx core.QueryContext, counterparty
 		return nil, err
 	}
 
+	defer logger.TimeTrack(now, "QueryUnfinalizedRelayPackets", "num_packets", len(packets))
```

Comment:
This `defer` is meaningless.

Reply:
The process is the same with or without defer now, but when the process is added after line 324 in the future, there is no need to worry about `logger.TimeTrack`, so I use defer.